react-backend/client/src/components/InternalServerError.js

```diff
@@ -1,15 +1,11 @@
-import React, { Component } from 'react';
+import React from 'react';
 
-export class InternalServerError extends Component {
-  render() {
-    return (
-      <div>
-        <h1>500 internal server error</h1>
-        <h2>
-          We are very sorry, but our server is throwing a tantrum. <br /> Our team of highly trained
-          monkeys is currently fixing this issue.
-        </h2>
-      </div>
-    );
-  }
-}
+export default () => (
+  <div>
+    <h1>500 internal server error</h1>
+    <h2>
+      We are very sorry, but our server is throwing a tantrum. <br /> Our team of highly trained
+      monkeys is currently fixing this issue.
```

Comment:
Not an actual problem but, I'd be up for changing this away from the google default maybe something like:

> Our highly talented and unusually attractive team is currently fixing this issue

Reply:
@RogeredBacon Can we get this copy change added to a new issue?